### DISCUSSION TITLE
openDCIM-23.04 Released!

### DISCUSSION THREAD

**Discussion by samilliken** [opendcim/openDCIM#1462] [Announcements]
Change Log for 23.04
•	Resolved UI issue with patch panel entry deletions
•	Resolved UI issue with patch panel editing
•	Corrected SQL in in new installs where admin user was not being created
•	Resolved issue where items that were disposed, then moved back to production, they could not be disposed again
•	Resolved UI issue where image could not be selected for template
•	Resolved some missed API routes causing images to not display in cabinets
•	Resolved issue with Outage Simulator Report when selecting subpanels

**Comment by Tarot67** [comment]
Great!. Fantastique!

**Comment by DannielleLumen** [comment]
One thing I'm seeing is that when I delete a device, it is not removing it from the rack.   But what it did was it removed all the options to update, delete, copy, Export Connections, Print, and Log View buttons.  As well as the Return to Navigator.   We were able to click and drag it to a different RU.  Is this a new feature or a bug?

**Comment by wilpig** [comment]
did you delete it from another tab?

From: Gwenster1 ***@***.***>
Date: Thursday, July 16, 2026 at 8:49 AM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [opendcim/openDCIM] openDCIM-23.04 Released! (Discussion #1462)


One thing I'm seeing is that when I delete a device, it is not removing it from the rack. But what it did was it removed all the options to update, delete, copy, Export Connections, Print, and Log View buttons. As well as the Return to Navigator. We were able to click and drag it to a different RU. Is this a new feature or a bug?

—
Reply to this email directly, view it on GitHub<https://github.com/opendcim/openDCIM/discussions/1462?email_source=notifications&email_token=AAMTY6U4LHJWC4ES5BO5VFT5FDMORA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZWGYZDEOBXUZZGKYLTN5XKU43VMJZWG4TJMJSWJJLFOZSW45FMMZXW65DFOJPWG3DJMNVQ#discussioncomment-17662287>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAMTY6VKUCA2OX4YHHR3FFT5FDMORAVCNFSNUABFKJSXA33TNF2G64TZHM2DANJZGE3TEO2ENFZWG5LTONUW63R3GU3TGNZUGAY2C5QC>.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS<https://github.com/notifications/mobile/ios/AAMTY6V52HNZDO4PRY7EZTD5FDMORA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZWGYZDEOBXUZZGKYLTN5XKU43VMJZWG4TJMJSWJJLFOZSW45FKMZXW65DFOJPWS33T> and Android<https://github.com/notifications/mobile/android/AAMTY6UI773FHQOZN6VKRXL5FDMORA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZWGYZDEOBXUZZGKYLTN5XKU43VMJZWG4TJMJSWJJLFOZSW45FOMZXW65DFOJPWC3TEOJXWSZA>. Download it today!

**Reply by DannielleLumen** [reply]
Yes, I opened the device in another tab.  Hit Delete.  Then went back to the Rack View Tab and hit refresh.  After hitting Refresh the device was still there.  Thinking I might have done something wrong, I tried it again, and when the device opened in another tab, all the buttons were gone.

**Reply by DannielleLumen** [reply]
I also just noticed that all users under this latest version were given additional rights.  And when I tried to update their permissions, it is now forcing me to add their email address.  How can I avoid this far too many folk don't have their emails in the tool.

**Reply by samilliken** [reply]
I need to finish wrapping up the 26.01 release, which will be my last one.   That has lots of bug fixes in it.   As for your user issue, I think we may have disabled the email requirement since 23.04.
